trimesh_hurricane.ipynb

```diff
@@ -0,0 +1,1237 @@
+    "We want to calculate the maximum water level at every cell over all time steps"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ds['zeta'].max(dim='time')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Dask will read data only when necessary, so nothing has been loaded yet. In the next cell, however, we use `persist` to persist the data to the Dask worker nodes, and watch the progress"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "max_var = ds['zeta'].max(dim='time').persist()\n",
+    "progress(max_var)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Visualize data on mesh using Datashader"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import datashader as dshade\n",
+    "import holoviews as hv\n",
+    "import geoviews as gv\n",
+    "import cartopy.crs as ccrs\n",
+    "\n",
+    "from holoviews.operation.datashader import datashade, rasterize\n",
+    "from matplotlib.cm import jet\n",
+    "\n",
+    "datashade.precompute = True\n",
+    "\n",
+    "hv.extension('bokeh')\n",
+    "%opts Image RGB VectorField [width=800 height=600]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "warnings.filterwarnings(\"ignore\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "v = np.vstack((ds['x'], ds['y'], max_var)).T\n",
+    "verts = pd.DataFrame(v, columns=['x','y','vmax'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "points = gv.operation.project_points(gv.Points(verts, vdims=['vmax']))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tris = pd.DataFrame(ds['element'].values.astype('int')-1, columns=['v0','v1','v2'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tiles = gv.WMTS('https://maps.wikimedia.org/osm-intl/{Z}/{X}/{Y}@2x.png')\n",
+    "value = 'max water level'\n",
+    "label = '{} (m)'.format(value)\n",
+    "trimesh = gv.TriMesh((tris, points), label=label)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "%%opts Image [colorbar=True] (cmap=jet)\n",
```

Comment:
yeh - I hate to sound like a moaner, but I agree with @jacobtomlinson about the "jet" colourmap. I'd be much happier with viridis if it's all the same to you https://www.climate-lab-book.ac.uk/2014/end-of-the-rainbow/